fe/fe-core/pom.xml

```diff
@@ -36,7 +36,7 @@ under the License.
 
     <properties>
         <doris.home>${basedir}/../../</doris.home>
-        <fe_ut_parallel>${env.FE_UT_PARALLEL}</fe_ut_parallel>
+        <fe_ut_parallel>1</fe_ut_parallel>
```

Comment:
Why?

Reply:
This can avoid the problem that intilj will prompt a pom syntax error when the environment variable is not set